src/webview/css/css.js

```diff
@@ -6,6 +6,7 @@ import cssNight from './cssNight';
 
 export default (theme: ThemeName) => `
 <link rel='stylesheet' type='text/css' href='./base.css'>
+<link rel='stylesheet' type='text/css' href='./katex/katex.min.css'>
```

Comment:
This is the only line of any part of this commit — summary, message, or diff — that I can reasonably be said to have written.

Please adjust its `Author:` field accordingly.

Reply:
OK -- changed.

Here's why I left your `Author:` line in place, though: this commit incorporates your work determining
* we need the CSS but not the JS, despite the install instructions just saying to use both
* we need the WOFF2 and not the other font formats, and the browser support details that make that the case
* the choice of where and how to lay out the KaTeX files

So my inclination if marking myself as author would be to have a `Co-authored-by:` line for you. I've left that out in this revision, because that feels most consistent with the spirit of your request here, and I want to respect your preferences on whether to claim this authorship. But I'd be glad to add such a line if you're OK with that.

Reply:
For the record, very belatedly: I do indeed feel that co-authorship would not have been appropriate here.

What I would probably write in a similar case would be something along the lines of "&zwj;(Based in part on/Supersedes/Inspired by) earlier work by...", with no pseudoheader. (As a concrete example, see the commit message of 988736b9862542aa9f9e6ca8e3ebe07bb52fc95b.)